app/models/manageiq/providers/storage_manager/storage_manager_mixin.rb

```diff
@@ -1,4 +1,4 @@
-module StorageManagerMixin
+module ManageIQ::Providers::StorageManager::StorageManagerMixin
```

Comment:
@jerryk55 And if we implement a manager for Amazon EBS, what would the Amazon provider include? `StorageManagerMixin`? no, that wouldn't apply, it would have to be `EbsManagerMixin`.